styleguide/source/_patterns/03-organisms/masthead.twig

```diff
@@ -1,8 +1,8 @@
-{%  set link, heading, subtitle, social = masthead.link, masthead.heading, masthead.subtitle, masthead.social %}
+{%  set indented, link, heading, subtitle, social = masthead.indented ? "indented" : "", masthead.link, masthead.heading, masthead.subtitle, masthead.social %}
 
 <header class="ama__masthead">
   <div class="ama__masthead__sidebar" role="complementary"></div>
-  <div class="ama__masthead__content" role="complementary">
+  <div class="ama__masthead__content {{ indented }}" role="complementary">
```

Comment:
We should be creating a semantic class here that matches our standards `ama__masthead--indented` instead of just `indented`.